### ISSUE TITLE
Build fails with `optimizationError` under pngquant

### ISSUE THREAD

**Issue by samrap** [roots/sage#1780]
## Submit a feature request or bug report

- [x] I've read the [guidelines for Contributing to Roots Projects](https://github.com/roots/guidelines/blob/master/CONTRIBUTING.md)
- [ ] This is a feature request
- [x] This is a bug report
- [x] This request isn't a duplicate of an [existing issue](https://github.com/roots/sage/issues)
- [x] I've read the [docs](https://roots.io/sage/docs) and [NPM Debugging Guidelines post](https://discourse.roots.io/t/npm-debugging-guidelines-failed-npm-install-bower-install-or-gulp-build-read-this/3060) and followed them (if applicable)
- [x] This is not a personal support request that should be posted on the [Roots Discourse](https://discourse.roots.io/c/sage) forums

Replace any `X` with your information.

---

**What is the current behavior?**

When running the build task, the process fails with an `optimizationError` while compressing images. The failure occurs when running [pngquant](https://github.com/pornel/pngquant). This appears to only be in issue on Linux systems, specifically CentOS (have not tested on Debian-based systems).

**What is the expected or desired behavior?**

When running a build task, the build should run successfully assuming all linting and syntax checks pass.

---

## Bug report

(delete this section if not applicable)

**Please provide steps to reproduce, including full log output:**

Running

`npm run build`

or 

`npm run build:production`

Will result in output similar to the following on CentOS/Linux systems:

```
94% asset optimizationError: Command failed: /home/username/public_html/web/app/themes/my-theme/node_modules/pngquant-bin/vendor/pngquant --output /tmp/e884b579-da68-4d8d-afec-7f203a306504 /tmp/7aeb9720-5172-4548-aae6-5330cc4d7226 --quality 65-90 --speed 4

    at /home/username/public_html/web/app/themes/my-theme/node_modules/execa/index.js:170:11
    at process._tickCallback (node.js:385:9)
error Command failed with exit code 1.
```

**Please describe your local environment:**

WordPress version: 4.6

OS: Yosemite

NPM/Node version: Node: 4.6.0 NPM 3.10.8

**Where did the bug happen? Development or remote servers?**

Remote servers. Remote server information:

- CentOS 7
- Node v4.7.0
- NPM v2.15.11
- Yarn v0.18.1

**Other relevant information:**

A similar bug can be found here: https://github.com/tcoopman/image-webpack-loader/issues/45

Looks like switching to `img-loader` solved the issue. However, according to [this issue](https://github.com/tcoopman/image-webpack-loader/issues/42), it doesn't solve the problem, but at leasts prevents the build task from failing.

More relevant information from @callmecavs on the second issue listed:

> this error has something to do with the pngquant quality setting, as lowering that - from 90 to 80 - fixed this issue for me.
My guess is a high quality setting leads to pngquant taking forever, which leads to execa timing out and throwing an error prematurely.
Updated: running the build in a CI setup caused it to fail with quality at 80. likely because the vm is slower.

For me, [changing Sage's pngquant config](https://github.com/roots/sage/blob/master/assets/build/webpack.config.optimize.js#L20) to as low as 40 on a 8 gig of RAM server _finally_  fixed the issue. 

Not sure if anyone else has come across this. As stated, everything runs fine on my local Mac environment. It is only once deploying to a development or production server which is CentOS based, does this error occur.

**Comment by QWp6t** [comment]
This isn't a bug in Sage, though. The issue should either be taken upstream or, as you mentioned, just lower the quality.

Also, for what it's worth, I don't think this is OS-specific. I think it more likely has to do with the system resources that are available. I used to run into this issue all the time with Sage 8 on my old machine; and again, lowering the quality resolved the issue.

The solution here is to make sure the setting is documented, which can be done in https://github.com/roots/docs

**Comment by samrap** [comment]
Circling back to this I honestly don't think lowering the quality is a valid solution, especially when you have to lower it to more than 50% of the original quality. I agree this is an issue with pngquant, but they do not seem interested in addressing the bug and there are other image optimization packages available so should we not consider switching libraries?

**Comment by QWp6t** [comment]
You can use whichever [imagemin plugins](https://www.npmjs.com/browse/keyword/imageminplugin) you want ([specify them in webpack.config.optimize.js](https://github.com/roots/sage/blob/master/assets/build/webpack.config.optimize.js#L22)). imagemin will use whichever one yields the smallest file. Right now we're using pngquant and optipng for png files. Set pngquant to `null` if you want to skip it entirely, and then it won't be attempted at all, and you'll only see compression done by optipng. But like I said, you can add any others you want.
